learn/advanced/filtering_and_faceted_search.md

```diff
@@ -220,6 +224,12 @@ You can use this filter when searching for `Planet of the Apes`:
 
 <CodeSamples id="filtering_guide_3" />
 
+`NOT director = "Tim Burton"` will include both documents that do not contain `"Tim Burton"` in its `director` field and documents without a `director` field. To return only documents that have a `director` field, expand the filter expression with the `EXISTS` operator:
```

Comment:
I think you should move this to 217, as we're saying two different things. But then the code sample won't be accurate. How about we get rid of the code sample for now and explain with just SQL?

Reply:
That makes sense 👍